app/src/main/res/values/strings.xml

```diff
@@ -98,4 +98,6 @@
     <string name="msg_no_permission">No permission</string>
     <string name="msg_edit_success">You have successfully edit this booking</string>
     <string name="msg_cant_find_date">Can\'t find booking day</string>
+    <string name="title_infomation">Infomation</string>
```

Comment:
WARNING: [Typos] "Infomation" is a common misspelling; did you mean "Information" ?
 This check looks through the string definitions, and if it finds any words that look like likely misspellings, they are flagged.